pillarbox-core-business/src/main/java/ch/srgssr/pillarbox/core/business/SRGMediaItem.kt

```diff
@@ -130,28 +143,33 @@ class SRGMediaItemBuilder internal constructor(mediaItem: MediaItem) {
     }
 
     /**
-     * Set force location
+     * Forces the location for IL/SAM backend calls.
      *
-     * @param forceLocation The location to use on the IL/SAM backend calls. Can be `null`, `CH`,  or `WW`.
+     * @param forceLocation The location to force. Valid values are:
+     *  - `null`: disables forced location and uses automatic detection.
+     *  - `"CH"`: forces the location to Switzerland.
+     *  - `"WW"`: forces the location to Worldwide.
      */
     fun forceLocation(forceLocation: String?) {
         this.forceLocation = forceLocation
     }
 
     /**
-     * Set vector
+     * Sets the vector.
      *
-     * @param vector The vector to forward to the integration layer.
-     * Should be [Vector.TV] or [Vector.MOBILE].
+     * @param vector The vector to forward to the integration layer. Should be either [Vector.MOBILE] or [Vector.TV].
      */
     fun vector(vector: String) {
```

Comment:
Same here. You have `Vector`, why are you using a `String` instead and documenting valid values?

Reply:
I've created a draft issue for that too.